bundles/CoreBundle/Resources/migrations/Version20191230104529.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+use Pimcore\Model\DataObject\ClassDefinition;
+
+class Version20191230104529 extends AbstractPimcoreMigration
+{
+    public function doesSqlMigrations(): bool
+    {
+        return false;
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $class = ClassDefinition::getById('EF_OSO');
```

Comment:
I would go by class name ... since old installations might have numeric class ids for onlineshoporder class.